README.md

```diff
@@ -77,4 +87,68 @@ $ singularity build osgvo-pilot.sif docker://opensciencegrid/osgvo-docker-pilot
 ```
 
 
+## CVMFS Without a Bind-Mount Using cvmfsexec
+
+If you don't have CVMFS available on the host, the container can still make
+CVMFS available by using [cvmfsexec](https://github.com/cvmfs/cvmfsexec#readme).
+
+You will need to specify a list of CVMFS repos to mount in the environment
+variable `CVMFSEXEC_REPOS`.
+
+On EL7, you must have kernel version >= 3.10.0-1127 (run `rpm -q kernel` to check),
+and user namespaces enabled.  On EL8, you must have kernel version >= 4.18.
```

Comment:
For user namespaces, maybe we should point people to step 1 here https://opensciencegrid.org/docs/worker-node/install-singularity/#enabling-unprivileged-singularity?